cloud-init/write-terraformer-aws-config.tpl.sh

```diff
@@ -4,7 +4,7 @@
 #
```

Comment:
Since I'm assuming the VNC username bits above were sourced from this script does it also apply here?

Reply:
Please see commit d9b1e38.